chains/opbnbtestnet/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://opbnb-testnet.bscscan.com/api
+    family: etherscan
+    name: opbnbscan
+    url: https://opbnb-testnet.bscscan.com/
+chainId: 5611
+deployer:
+  name: JunayedMh
+  url: https://github.com/JunayedMh
+displayName: Op Bnb Testnet
+domainId: 5611
+isTestnet: true
+name: opbnbtestnet
+nativeToken:
+  decimals: 18
+  name: Binance
+  symbol: tBNB
+protocol: ethereum
+rpcUrls:
+  - http: https://opbnb-testnet-rpc.publicnode.com
+    http: https://opbnb-testnet-rpc.bnbchain.org
```

Comment:
One last change. This should be:
 ```yaml
 rpcUrls:
  - http: https://opbnb-testnet-rpc.publicnode.com
  - http: https://opbnb-testnet-rpc.bnbchain.org
 ```

Because array elements in yaml are separated by `-`